### ISSUE TITLE
Allow numbers as tag IDs

### ISSUE THREAD

**Issue by vegeta897** [jprochazk/uecs#2]
It would be convenient if tags accepted numbers as well as strings, so that TS enums could be used like this:

```ts
export enum TagID {
  Turret,
  Spawner,
  Bullet,
}
```

... instead of like this:

```ts
export enum TagID {
  Turret = 'turret',
  Spawner = 'spawner',
  Bullet = 'bullet',
}
```

**Comment by jprochazk** [comment]
Indeed, this can be easily done. The way the Tag system works is that it stringifies the input to the `Tag.for` method and uses that as the unique identifier for the Tag, so anything that has the `.toString()` method will work, even an object. I'll release a new version today with this update 🙂

**Comment by jprochazk** [comment]
Fixed in 77233a7.

**Comment by jprochazk** [comment]
Included this in release v0.3.0.
